### PR TITLE
Update pawsey_nimbus.md

### PR DESCRIPTION
**Pull request by georgiesamaha**
Corrected c2r8 RAM from 4 GB to 8 GB

---
name: New Config
about: A new cluster config
---

Please follow these steps before submitting your PR:

- [ ] If your PR is a work in progress, include `[WIP]` in its title
- [ ] Your PR targets the `master` branch
- [ ] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [ ] Add your custom config file to the `conf/` directory
- [ ] Add your documentation file to the `docs/` directory
- [ ] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [ ] Add your custom profile to the `README.md` file in the top-level directory
- [ ] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

**Comment by georgiesamaha** [comment]
@nf-core-bot fix linting pretty please

**Review by maxulysse — APPROVED** [review]

**Comment by maxulysse** [comment]
@nf-core-bot fix linting pretty please 🙏

**Comment by maxulysse** [comment]
@nf-core-bot fix linting pretty please 🙏

File: docs/pawsey_nimbus.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Currently profiles for various [instance flavours](https://support.pawsey.org.au
 
 Profiles for various standard instance flavours are available in this config. Please select a profile based on the flavour (i.e. CPUs, and RAM it has access to):
 
-- c2r8: 2 CPU cores, 4Gb RAM
+- c2r8: 2 CPU cores,  8 Gb RAM
```
Comment (jfy133) [inline docs/pawsey_nimbus.md:13]:
```suggestion
- c2r8: 2 CPU cores, 8 Gb RAM
```

**Comment by jfy133** [comment]
@georgiesamaha the commit above should fix the issue :)

**Comment by georgiesamaha** [comment]
Thanks @jfy133, but I'm having issues accepting your suggested change or pushing my own edits. I'm going to close this PR and try again. Sorry for the inconvenience.

**Pull request closed without merging**